### PR TITLE
Pai cards flash and play a noise when a ghost submits

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
The pai cards now play a small animation when a ghost submits themself as a personality, as well as chiming

## Why It's Good For The Game
Its easy to miss when ghosts submit themself as a personality, only sending a chat message. With this change, its more noticable and ghosts are more likely to get back into the round

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/73374039/188325381-0e79622c-dfc7-4e8a-b431-1c81c858870a.mp4

</details>

## Changelog
:cl:
tweak: pai cards chime when new personalities are submitted
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/paicard.dm [hunk 1]
```diff
@@ -168,6 +168,9 @@
 				src.add_overlay("pai-sunglasses")
 
 /obj/item/paicard/proc/alertUpdate()
+	var/image/I = image(icon, src, icon_state = "pai-alert")
+	flick_overlay_view(I, src, 5 SECONDS)  // overlay workaround
```
Comment (Penwin0) [inline code/game/objects/items/devices/paicard.dm]: How is this a workaround? I don't think this comment is necessary, otherwise PR is good

Comment (etherware-novice) [reply code/game/objects/items/devices/paicard.dm]: i spent a whole hour trying to figure out how overlays worked

Comment (Penwin0) [reply code/game/objects/items/devices/paicard.dm]: This is fine, just remove the comment and I'll approve

**Review by Penwin0 — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Can you spam this to hell?

**Comment by etherware-novice** [comment]
well can you spam the "the card flashes a notification" message it just reuses that proc

**Comment by Penwin0** [comment]
It's not spammable no, I checked

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**